app/forms/core_induction_programme_choice_form.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CoreInductionProgrammeChoiceForm
+  include ActiveModel::Model
+
+  attr_accessor :core_induction_programme_id
+
+  validates :core_induction_programme_id, presence: { message: "Select one" }
```

Comment:
I have the copy here from Shani
```suggestion
  validates :core_induction_programme_id, presence: { message: "Select the training materials you want to use" }
```